src/test/java/de/tum/in/www1/artemis/ApollonDiagramResourceIntegrationTest.java

```diff
@@ -90,6 +90,35 @@ public void testGetAllApollonDiagram_OK() throws Exception {
         assertThat(updatedResponse.size()).as("updated response has length 2").isEqualTo(2);
     }
 
+    @Test
```

Comment:
In general you might test here that the evaluation of the courseId actually works. E.g. add two diagrams with different courses to the repository and test that only the one with the matching course is returned.

Reply:
As discussed.